odrivemotor/src/odriveSerial/odriveSerial.py

```diff
@@ -77,6 +77,8 @@ def reconfigure(self, config: ComponentConfig, dependencies: Mapping[ResourceNam
             self.current_lim = self.odrv.axis0.config.general_lockin.current
 
     async def set_power(self, power: float, extra: Optional[Dict[str, Any]] = None, **kwargs):
+        if abs(power) < 0.1:
```

Comment:
are the Canbus errors being less than `.0001` and serial less than `0.1` intentional?

Reply:
oops no, I'll fix it

Reply:
this will be the only thing not updated in the Darwin version, but I'm personally fine with leaving it as is and changing it next time there needs to be odrive changes as long as you're not opposed to that